template/src/lib/ui/widgets/HistogramWidgetUI.js

```diff
@@ -40,13 +40,25 @@ function __generateDefaultConfig(
 ) {
   return {
     grid: {
-      left: 8,
-      top: 20,
-      right: 8,
-      bottom: 60,
+      left: 0,
+      top: 16,
+      right: 0,
+      bottom: 24,
+    },
+    axisPointer: {
+      lineStyle: {
+        color: theme.palette.charts.axisPointer,
+      },
     },
     tooltip: {
       trigger: 'axis',
+      padding: [4, 8],
+      textStyle: {
+        ...theme.typography.caption,
+        fontSize: 12,
+        lineHeight: 16,
```

Comment:
Is it a custom typography?

Reply:
No, but Echarts doesn't support sizes as em, rem

Reply:
:+1: